doc/manuals/admin/management_api.md

```diff
@@ -1,5 +1,6 @@
 # Management REST interface
 
+## Log and Trace levels
```

Comment:
In my opinion, "Statistics" have enough conceptual integrity to remain in its own chapter. In addition, moving it have some impact in "doc backward compatibility" (as user I except to find statistics information in the same place where I have found it during months).

By the moment, I'd leave things as they are now... although of course in the future we may change.